src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Resources/CSharpMarkAssembliesWithNeutralResourcesLanguage.cs

```diff
@@ -15,16 +15,17 @@ namespace Microsoft.NetCore.CSharp.Analyzers.Resources
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public sealed class CSharpMarkAssembliesWithNeutralResourcesLanguageAnalyzer : MarkAssembliesWithNeutralResourcesLanguageAnalyzer
     {
-        protected override void RegisterAttributeAnalyzer(CompilationStartAnalysisContext context, Action onResourceFound)
+        protected override void RegisterAttributeAnalyzer(CompilationStartAnalysisContext context, Action onResourceFound, INamedTypeSymbol generatedCode)
         {
-            context.RegisterSyntaxNodeAction(nc =>
+            context.RegisterSyntaxNodeAction(context =>
             {
-                if (!CheckAttribute(nc.Node))
+                var attributeSyntax = (AttributeSyntax)context.Node;
```

Comment:
Previous code was having the possibility that the `Node` isn't a `AttributeSyntax`, it also doesn't look possible but I am wondering if that was done for a specific reason or just by "node is generic so let's work in the generic case".

Reply:
@Evangelink It should always be `AttributeSyntax` since we do `RegisterSyntaxNodeAction(...., SyntaxKind.Attribute)`